go/vt/vtgate/tabletgateway.go

```diff
@@ -36,6 +36,7 @@ import (
 	"vitess.io/vitess/go/vt/vtgate/buffer"
 	"vitess.io/vitess/go/vt/vttablet/queryservice"
 
+	"vitess.io/vitess/go/vt/proto/query"
```

Comment:
Looks like this is already imported on line 40 as "querypb"

Reply:
ahh nice catch

Reply:
updated